tests/linalg/svd.rs

```diff
@@ -38,6 +39,7 @@ mod proptest_tests {
                         prop_assert!(relative_eq!(m, &u * ds * &v_t, epsilon = 1.0e-5));
                         prop_assert!(u.is_orthogonal(1.0e-5));
                         prop_assert!(v_t.is_orthogonal(1.0e-5));
+                        prop_assert!(s.as_slice().windows(2).all(|elts| elts[0] >= elts[1]));
```

Comment:
might be nice to have a utility method `is_sorted(slice)`..

Reply:
Done.
Actually, `slice.is_sorted` already exists: https://doc.rust-lang.org/std/primitive.slice.html#method.is_sorted
But it hasn’t been stabilized yet. Also, it checks that it’s sorted in ascending order (though we could use `is_sorted_by` instead).